internal/cache/shadowdb_test.go

```diff
@@ -89,9 +90,9 @@ func TestNextShadowEntry(t *testing.T) {
 	}
 
 	cacheDir := t.TempDir()
-	insertUsersInDb(t, cacheDir)
+	testutils.PrepareDBsForTests(t, cacheDir, "users_in_db")
 
-	c := newCacheForTests(t, cacheDir, cache.WithTeardownDuration(0), cache.WithOfflineCredentialsExpiration(0))
+	c := testutils.NewCacheForTests(t, cacheDir)
```

Comment:
This was called with the 2 options which are not in `NewCacheForTests()`, I know we don’t need `cache.WithOfflineCredentialsExpiration` anymore with "user_in_db", but the teardown duration to 0 is better.